hicommonwealth/commonwealth
# Token launcher for home page

## Link to Issue
Closes: #10266

## Description of Changes
- Adds the Token Launcher component in Community Home.
- Implements functionality for users to input ideas for tokens they want to launch.

## "How We Fixed It"


## Test Plan
- Navigate to any community.
- Select Community Home tab.
- Verify that token launcher components is functional.

## Deployment Plan

1. 

## Other Considerations

-
